packages/atomic/src/components/commerce/atomic-commerce-product-list/e2e/atomic-commerce-product-list.e2e.ts

```diff
@@ -15,16 +15,6 @@ test.describe('when no first search has yet been executed', async () => {
   }) => {
     await expect(productList.placeholders.nth(23)).toBeVisible();
   });
-
-  test('should open product in the same tab by default', async ({
```

Comment:
@y-lakhdar I removed this test because it doesn't seem to make sense under this describe: if no search has been executed, there should only be placeholders, so products can't be opened.